style.css

```diff
@@ -337,3 +331,35 @@ header .links li:first-child {
   background-repeat: no-repeat;
   padding: .5em;
 }
+
+
+@media (max-width: 650px) {
+
+  header .links {
+    display: none;
+  }
+
+  .header, .footer {
+    background-color: #0c3757;
+    color: white;
+  }
+
+  main {
+    margin: 0;
+    flex-direction: column;
+  }
+
+  .sidebar {
+    width: 100%;
+    margin: 0;
```

Comment:
Without the various margins around the sidebar and .main boxes it looks a bit strange. The rounded border corners don't look great, and the 1px borders around each box touch eachother and end up forming 2px borders.
I think we need a bit of UI polish here and there to make it look a bit better.

Also, I'm unsure about the where the sidebar with the list of tools should go. I think the list of bugs is the most important part here, so it should go first (so maybe `order:-1;` on `.main`) , but then users would have to scroll for a long time to find the list of tools.

So maybe the right fix is to leave the list of tools first at the top, but make it smaller? Or even partly hidden with a "show more" button that reveals the full list. 
If Inspector, Web Console, JS Debugger and Network Monitor are the only ones visible at first, I think we're fine, because those are the most used tools.

One simple thing would be to add
```
.filters {
	display: flex;
	flex-wrap: wrap;
}
```
which would at least display the various inputs and their labels on lines, rather than vertically.